pipeline/ingest/tracking.py

```diff
@@ -183,12 +183,14 @@ def make(self, key):
                         **{fmap[k]: v for k, v in recs['paw_right'].items()
                            if k in fmap}}, allow_direct_insert=True)
 
-                TrackingIngest.TrackingFile.insert1(
-                    {**key, 'trial': tmap[t], 'tracking_device': tdev,
+                tracking_files.append({**key, 'trial': tmap[t], 'tracking_device': tdev,
                      'tracking_file': str(tfull.relative_to(tdat))})
 
             log.info('... completed {}/{} items.'.format(i, n_tmap))
 
+        self.insert1(key)
+        self.TrackingFile.insert(tracking_files)
```

Comment:
was this done just to speed up the trackingfile.insert1 or for some other reason?